scripts/e2e-oie.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+export CHROME_HEADLESS=true
+setup_service google-chrome-stable 87.0.4280.66-1
+export OIE_TESTS=true
+
+source $OKTA_HOME/$REPO/scripts/setup.sh
+
+export TEST_SUITE_TYPE="junit"
+export TEST_RESULT_FILE_DIR="${REPO}/build2/reports/junit"
+echo $TEST_SUITE_TYPE > $TEST_SUITE_TYPE_FILE
+echo $TEST_RESULT_FILE_DIR > $TEST_RESULT_FILE_DIR_FILE
+
+# This file contains all the env vars we need for e2e tests
+aws s3 --quiet --region us-east-1 cp s3://ci-secret-stash/prod/signinwidget/export-test-credentials.sh $OKTA_HOME/$REPO/scripts/export-test-credentials.sh
+source $OKTA_HOME/$REPO/scripts/export-test-credentials.sh
+
+# We use the below OIE enabled org and clients for OIE tests
```

Comment:
Shall we create a new file to s3 and merge those config to it?

Reply:
The variables are the same in both the orgs. It's easier to have a single file I think.

Reply:
I meant those 3 variables below. they shall be different with non-OIE org?

Reply:
We can't merge both into the same config, because these 3 variables would be different for OIE and non-OIE orgs.
I can add these in a separate config file in S3 but these variables are not secret. Do we need them in S3?

Reply:
Feel it's better to in s3 file as well, just be consistent?